src/node/models/PromiseWithStd.ts

```diff
@@ -0,0 +1,8 @@
+import {Readable} from "node:stream";
+import {ChildProcess} from "node:child_process";
```

Comment:
If you import something as type use this syntax:

```suggestion
import type {Readable} from "node:stream";
import type {ChildProcess} from "node:child_process";
```

this avoids confusion for junior devs. 

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html

Reply:
Thanks for the suggestion. I fixed that.